extensions/websockets-next/deployment/src/main/java/io/quarkus/websockets/next/deployment/WebsocketTelemetryCustomizerBuildItem.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.websockets.next.deployment;
+
+import java.util.function.Consumer;
+
+import io.quarkus.builder.item.MultiBuildItem;
+import io.quarkus.websockets.next.runtime.telemetry.WebsocketTelemetryProviderBuilder;
+
+/**
+ * Provides a way to set up metrics and/or traces support in the WebSockets extension.
+ */
+final class WebsocketTelemetryCustomizerBuildItem extends MultiBuildItem {
```

Comment:
I wonder if this build item is actually needed. I mean, `TracesBuilderCustomizer` could be a CDI bean (conditionally registered) and the `WebsocketTelemetryProvider` synthetic bean can add a [synthetic injection point](https://quarkus.io/guides/cdi-integration#synthetic-injection-points) for `Consumer<WebsocketTelemetryProviderBuilder>`.